packages/valory/services/trader_pearl/service.yaml

```diff
@@ -126,7 +126,7 @@ type: connection
 config:
   ledger_apis:
     ethereum:
-      address: http://host.docker.internal:8545
+      address: ${GNOSIS_LEDGER_RPC:str:http://host.docker.internal:8545}
       chain_id: 100
       default_gas_price_strategy: eip1559
       poa_chain: false
```

Comment:
please also override these ones.

Reply:
see for example https://github.com/valory-xyz/optimus/blob/main/packages/valory/services/optimus/service.yaml#L113-L116